tests/data/README.md

```diff
@@ -411,6 +411,14 @@ Box structure of the items in this file:
 [alpha] [alpha]
 ```
 
+### File [alpha_noispe.avif](alpha_noispe.avif)
+
+License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LICENSE)
+
+Source: Same pixels as `color_grid_alpha_nogrid.avif` encoded with
+`avifenc --lossless`. Manually edited with a hex editor to remove the alpha
```

Comment:
Optional: We could encode the image with cavif-rs, with the fix to avif-serialize (https://github.com/kornelski/avif-serialize/issues/3) reverted. That was the encoder with this issue.